frontend/src/queries/schema.ts

```diff
@@ -326,7 +328,7 @@ export interface DataTableNode extends Node {
 
 // Saved insight node
 
-export interface SavedInsightNode extends Node, InsightVizNodeViewProps {
+export interface SavedInsightNode extends Node, InsightVizNodeViewProps, DataTableNodeViewProps {
```

Comment:
Is this change on purpose? The `SavedInsightsNode` does not support most of the `DataTableNodeViewProps` options.

Reply:
**tldr:** It was needed to support saved HogQL tables in notebooks. 

**Full explanation:**
Before the `<Query` component called from `SavedInsight` was constructed via
```
query={{ ...query, kind: NodeKind.InsightVizNode, source: filtersToQueryNode(insight.filters) }}
```
which assumed that saved insights were always of kind `InsightVizNode`.

For saved SQL table queries, the transformed kind on the [query generated by the insightDataLogic](https://github.com/PostHog/posthog/pull/17442/files#diff-4c99f694d9e48053fe2b76c4a044e218829722d68e7e10d95064f6de4d6cd149R20) was of type `DataTableNode`. In order to then render that node correctly & appease the linter I needed the `SavedInsightNode` to allow the view props on a table to be set. Done in https://github.com/PostHog/posthog/pull/17442/files#diff-c1c303fd13bcb91112ea40b294c4fd493f0e6c7085d5c481544aac3fee8ec2b0R36-R42